Makefile

```diff
@@ -11,6 +11,8 @@ clean: clean_openwrt clean_feeds clean_binaries
 
 # Building OpenWRT
 $(DIR__CI)/patched:
+	/vault read -field=key secret/creator/packagesigning > $(DIR__OPENWRT)/key.pem
```

Comment:
should we also remove the key.pem,cert.pem during clean rule? as we are removing the .patched file too?